bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableDataGrpcClient.java

```diff
@@ -80,7 +92,7 @@
  * @author sduskis
  * @version $Id: $Id
  */
-public class BigtableDataGrpcClient implements BigtableDataClient {
+public class BigtableDataGrpcClient implements BigtableDataClient, IBigtableDataClient {
```

Comment:
Please create a separate class that implements `IBigtableDataClient` and uses a `BigtableDataClient`